style.css

```diff
@@ -106,27 +127,91 @@ ul li:hover ul {
 	visibility: visible;
 }
 
+/* Text underline hover */

```

Comment:
Link hover animation was added for an aesthetic effect to attract the users attention and make the website a high quality outcome